student/src/main/webapp/Shared/CSS/Universal/shell.css

```diff
@@ -27,8 +27,8 @@ body {
 	overflow:hidden;
```

Comment:
It appears that this file was formatted outside of the areas where actual changes were made. These formatting changes make it very difficult to tell what actually changed. Please re-submit this file without the formatting.